backend/src/test/java/eu/bbmri_eric/negotiator/integration/service/NegotiationLifecycleServiceImplTest.java

```diff
@@ -169,18 +169,20 @@ void sendEvent_abandonNegotiation_to_inProcess_Negotiation() throws IOException
             negotiationDTO.getId(), NegotiationEvent.ABANDON, "Not acceptable"));
   }
 
-  private NegotiationDTO saveNegotiation() throws IOException {
-    NegotiationCreateDTO negotiationCreateDTO = TestUtils.createNegotiation("request-2");
+  private NegotiationDTO saveNegotiation(boolean draft) throws IOException {
```

Comment:
I would add it as an optional parameter, usually you wont save it as draft

Reply:
you mean by adding

```java
  private NegotiationDTO saveNegotiation() throws IOException {
    return saveNegotiation(false);
  }
```
?

Reply:
something like this yes